aepp/config.py

```diff
@@ -22,7 +24,8 @@
 
 # endpoints
 endpoints = {
-    "global": "https://platform.adobe.io",
+    # global endpoint is https://platform.adobe.io in prod, otherwise https://platform-$ENV.adobe.io
+    "global": "",
```

Comment:
Could we have something like globalprod and globalstage so centralized location for the endpoint instead of defining it manually afterwards for each connection ? 
Or would a stage environment is different per service ?

Reply:
There's also `int` environment which goes to `platform-int.adobe.io` and `qa` that goes to `platform-qa.adobe.io` (and maybe other environments I'm not aware of) so I made this dynamic to future-proof it and avoid defining 1 variable per env, let me know if that makes sense or if you'd like me to define separate variables for int, stage and qa.